boid_controller_node:
  ros__parameters:
    num_boids: 4
    adjacency:
      - [false, true,  true,  true]
      - [true,  false, true,  true]
      - [true,  true,  false, true]
      - [true,  true,  true,  false]
